<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgba(52, 177, 94, 0.281);</string>
   </property>
   <widget class="QPushButton" name="BtnCalc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba( 0)</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeMod">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lMod">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>45</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0);</string>
    </property>
    <property name="text">
     <string>Mod</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>45</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0);</string>
    </property>
    <property name="text">
     <string>Pow</string>
    </property>
   </widget>
   <widget class="QLabel" name="lBase">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>45</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0);</string>
    </property>
    <property name="text">
     <string>Base</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LePow">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeBase">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0)</string>
    </property>
    <property name="text">
     <string>Result by Montgomery: </string>
    </property>
   </widget>
   <widget class="QLabel" name="res">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>45</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0);</string>
    </property>
    <property name="text">
     <string>Modular exponentiation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0)</string>
    </property>
    <property name="text">
     <string>Result by usual modular exponentiation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="res_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>45</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(52, 177, 94, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
